client/src/components/Donation/DonateForm.js

```diff
@@ -255,8 +255,8 @@ class DonateForm extends Component {
           {subscriptionPayment ? (
             <Fragment>
               <b>
-                Confirm your donation of ${donationAmount / 100} / year with
-                PayPal:
+                Confirm your donation of ${donationAmount / 100} /{' '}
```

Comment:
I believe these lines can be simplified to the following:

```js
Confirm your donation of ${donationAmount / 100} / {donationDuration} with
PayPal:
```

Reply:
I just checked, it seems like linter does that on save.